src/.chezmoiignore

```diff
@@ -20,6 +20,9 @@ tools/**
 **/BUILD
 **/BUILD.bazel
 
+# Ignore oh-my-zsh cache
+/.oh-my-zsh/cache/
```

Comment:
can you explain this? why is it needed? is it for the dstination state? should consider templating it in as well